xml/System.Buffers.Text/Utf8Parser.xml

```diff
@@ -416,13 +542,26 @@
         <Parameter Name="standardFormat" Type="System.Char" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="value">To be added.</param>
-        <param name="bytesConsumed">To be added.</param>
-        <param name="standardFormat">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">The Utf8 string to parse.</param>
+        <param name="value">When the method returns, contains the value parsed from <paramref name="source" />, if the parsing operation succeeded. </param>
+        <param name="bytesConsumed">If the parsing operation was successful, contains the length in bytes of the parsed substring when the method returns. If the method fails, <paramref name="bytesConsumed" /> is set to 0.</param>
+        <param name="standardFormat">The expected format of the Utf8 string.</param>
+        <summary>Parses an <see cref="T:System.SByte" /> at the start of a Utf8 string.</summary>
+        <returns><see langword="true" /> for success; <see langword="false" /> if the string was not syntactically valid or an overflow or underflow occurred.</returns>
```

Comment:
We are using "string" here quite loosely (to mean UTF-8 bytes containing text data). Maybe its OK.